kubric/core/lights.py

```diff
@@ -31,6 +31,11 @@ def get_background_default(self):
     return True
 
 
+class SpotLight(Light):
+  spot_blend = tl.Float(1.)
+  spot_size = tl.Float(*(0.00, 180.00))
```

Comment:
`tl.Float` takes just the default value not the range. If you wish to add validation you'll need to add a `@validate('spot_size')` method. 
Also, Blender internally uses radians (not degrees), so the range would go up to `pi` not `180`. 
I suggest to just change this to `tl.Float(np.pi / 4)`  which sets the default to 45 degree (the default for a spotlight in Blender).